src/Desalt.Core/Translation/ExpressionTranslator.ExpressionVisitor.cs

```diff
@@ -209,6 +209,14 @@ public override ITsExpression VisitIdentifierName(IdentifierNameSyntax node)
                     return Factory.Identifier(node.Identifier.Text);
                 }
 
+                if (ScriptSymbolTable.TryGetValue(symbol, out IScriptSymbol? scriptSymbol))
```

Comment:
TryGetValue takes in a generic which will try to look up the symbol as well as attempt to cast to the type. So, if you use a IScriptTypeSymbol as the out parameter instead of the more-generic IScriptSymbol, you can do this in one line:

```suggestion
                if (ScriptSymbolTable.TryGetValue(symbol, out IScriptTypeSymbol? typeSymbol) && typeSymbol.TreatMethodsAsGlobal)
```

Reply:
nice!